SPECS/ocaml-sexplib/ocaml-sexplib.spec

```
@@ -0,0 +1,337 @@
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+%ifnarch %{ocaml_native_compiler}
+%global debug_package %{nil}
+%endif
+
+%bcond_with vim
+
+%global srcname sexplib
+
+Name:           ocaml-%{srcname}
+Version:        0.16.0
+Release:        1%{?dist}
+Summary:        Automated S-expression conversion
+
+# The project as a whole is MIT, but code in the src subdirectory is BSD.
+License:        MIT and BSD
+URL:            https://github.com/janestreet/%{srcname}
+Source0:        %{url}/archive/v%{version}/%{srcname}-%{version}.tar.gz
+
+BuildRequires:  ocaml >= 5.1.1
+BuildRequires:  ocaml-dune >= 2.0.0
+BuildRequires:  ocaml-num-devel
+BuildRequires:  ocaml-parsexp-devel >= 0.16
+BuildRequires:  ocaml-sexplib0-devel >= 0.16
+BuildRequires:  vim
+
+%if %{with test}
+BuildRequires:  ocaml-base-devel
+BuildRequires:  ocaml-base-quickcheck-devel
+BuildRequires:  ocaml-core-kernel-devel
+BuildRequires:  ocaml-expect-test-helpers-core-devel
+BuildRequires:  ocaml-ppx-jane-devel
+BuildRequires:  ocaml-sexp-grammar-validation-devel
+%endif
+
+%description
+This package contains a library for serializing OCaml values to and from
+S-expressions.
+
+%package        devel
+Summary:        Development files for %{name}
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+Requires:       ocaml-num-devel%{?_isa}
+Requires:       ocaml-parsexp-devel%{?_isa}
+Requires:       ocaml-sexplib0-devel%{?_isa}
+
+%description    devel
+The %{name}-devel package contains libraries and signature files for
+developing applications that use %{name}.
+
+%package        vim
+Summary:        Support for sexplib syntax in vim
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+Requires:       vim
+
+%description    vim
+This package contains a vim syntax file for Sexplib.
+
+%prep
+%autosetup -n sexplib-%{version}
+
+%build
+%dune_build
+
+%install
+%dune_install
+
+# Install the vim support
+mkdir -p %{buildroot}%{vimfiles_root}/syntax
+cp -p vim/syntax/sexplib.vim %{buildroot}%{vimfiles_root}/syntax
+
+%if %{with test}
+%check
+%dune_check
+%endif
```

```suggestion
%check
%dune_check
```

Please clean similar cases everywhere else.

---

Fixed